cliclient/cliclient_test.go

```diff
@@ -0,0 +1,101 @@
+package cliclient
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"net/http/httptest"
+	"net/url"
+	"os"
+	"reflect"
+	"runtime"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+
+	"github.com/rancher/cli/cliclient/mocks"
+	"github.com/rancher/cli/config"
+	"github.com/rancher/norman/types"
+)
+
+func TestCreateClientOpts(t *testing.T) {
```

Comment:
Can you move it to `Test_createClientOpts` in order to follow the tests naming convention?

Reply:
Done.